test/e2e/lib/typemetas.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lib
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+var JobTypeMeta = batchTypeMeta("Job")
+
+func batchTypeMeta(kind string) *metav1.TypeMeta {
+	return &metav1.TypeMeta{
+		Kind:       kind,
+		APIVersion: "batch/v1",
+	}
+}
+
+var KsvcTypeMeta = servingTypeMeta("Service")
+
+func servingTypeMeta(kind string) *metav1.TypeMeta {
+	return &metav1.TypeMeta{
+		Kind:       kind,
+		APIVersion: "serving.knative.dev/v1",
+	}
+}
+
+var ChannelTypeMeta = gcloudMessagingTypeMeta("Channel")
```

Comment:
why do we need `gcloud` prefix here, and all of the ones below... Is this coupled with gcloud?

Reply:
Removed `gcloud` prefix.